packages/labs/analyzer/src/lib/javascript/jsdoc.ts

```diff
@@ -15,120 +26,264 @@ import {AnalyzerInterface, NamedJSDocInfo, NodeJSDocInfo} from '../model.js';
  */
 
 /**
- * Remove line feeds from JSDoc summaries, so they are normalized to
+ * Returns true if given node has a JSDoc tag
+ */
+export const hasJSDocTag = (node: ts.Node, tag: string) => {
+  return ts.getJSDocTags(node).some((t) => t.tagName.text === tag);
+};
+
+/**
+ * Remove line feeds from JSDoc comments, so they are normalized to
  * unix `\n` line endings.
  */
-const normalizeLineEndings = (s: string) => s.replace(/\r/g, '');
+const normalizeLineEndings = (s: string) => s.replace(/\r/g, '').trim();
```

Comment:
Just "normalize" now that it trims?

Reply:
The first-order purpose is normalizing windows/unix line endings (i.e. getting `\r` out); the trim was just adding a little extra value-add. So I think keeping the "line endings" in the name makes it clearer what it's doing.